### PR TITLE
Serve documents on applications

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

**Comment by richtier** [comment]
@kerin the api returns 
```
{
     ...
     'url': ... # the s3 signed url
     ...
}
```

and the lite-internal-frontend is [responsible for redirecting to that url](https://github.com/uktrade/lite-internal-frontend/pull/580/files#diff-c3313fa08df669b358eccd5f1f46a95fR83).

Looks like I misunderstood the flow diagram from the ticket and had the frontend do the redirect. Having said that, this may be better than the endpoint returning 302 as I dont think I've seen an endpoint do a 302 before, but open to advisement :+1:

File: spire/serializers.py [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 import xmltodict
 
+from django.conf import settings
+
 from spire import models
 
 
+s3_client = boto3.client(
+    "s3",
+    config=Config(signature_version="s3v4"),
```
Comment (r4vi) [inline spire/serializers.py:13]: isn't this the default?

Comment (richtier) [reply spire/serializers.py:13]: oddly, yes. but when I left `config=...` out the url generated was not s3v4. I'll try again and see

Comment (richtier) [reply spire/serializers.py:13]: worked fine. will remove :+1:

**Pull request merged**